index.html

```diff
@@ -33,6 +33,14 @@
     />
 
     <!-- Custom -->
+    <link
+      rel="prefetch"
+      href="materials.json"
+      as="fetch"
+      type="text/json"
+      crossorigin="anonymous"
+      onload="fetch('materials.json').then(res=>res.json()).then(obj=>window.materials=obj)"
```

Comment:
This is too hacky and exposes the cell data in the global scope, revert back to regular object for now.

Reply:
What if I delete it right after Cell.js gets a reference to it?